tools/lib/test_server.py

```diff
@@ -20,14 +20,18 @@
     sys.path.insert(0, os.path.dirname(TOOLS_DIR))
 
 from zerver.lib.test_fixtures import update_test_databases_if_required
+from scripts.lib.zulip_tools import get_or_create_dev_uuid_var_path
 
 def set_up_django(external_host):
     # type: (str) -> None
     os.environ['EXTERNAL_HOST'] = external_host
     os.environ["TORNADO_SERVER"] = "http://127.0.0.1:9983"
+    os.environ["LOCAL_UPLOADS_DIR"] = "1"
```

Comment:
What's the story with setting this here?  I don't think it's necessary or useful, since `settings.py` does not read this value from the environment.

Reply:
So, that's what I was thinking, but simply setting the value for `LOCAL_UPLOADS_DIR` here doesn't work.  If `LOCAL_UPLOADS_DIR` is unconditionally set in `test_settings.py`, then it seems one can't change the value here.  Is there more appropriate way to achieve this?